apps/customer-portal/webapp/.agents/05-FEATURES-OTHER.md

```
@@ -0,0 +1,450 @@
+## 4. Project Details (`src/features/project-details/`)
+
+### Purpose
+Project profile: overview metadata/contacts/service hours, deployments (add/edit/delete environments and products), and time tracking (case time cards).
+
+### Routes
+```
+/projects/:projectId/project-details    ProjectDetailsPage (tab bar)
+```
+
+### Pages
+| Page | Tabs |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)